test-packages/type-tests/test/v4/filters.spec.ts

```diff
@@ -2,20 +2,37 @@ import {
   TestEnumType,
   testService
 } from '@sap-cloud-sdk/test-services-odata-v4/test-service';
-import { and } from '@sap-cloud-sdk/odata-common/internal';
-import { any } from '@sap-cloud-sdk/odata-v4';
 
-const { testEntityApi, testEntityMultiLinkApi } = testService();
+const { testEntityApi } = testService();
 
 const schema = testEntityApi.schema;
-const multiLinkSchema = testEntityMultiLinkApi.schema;
-
-// $ExpectType FilterList<TestEntity<DeSerializers<string, boolean, number, BigNumber, number, number, number, number, BigNumber, string, number, number, string, any, Moment, Moment, Duration, Time, any>>, DeSerializers<string, boolean, number, BigNumber, number, number, number, number, BigNumber, string, number, number, string, any, Moment, Moment, Duration, Time, any>>
-and(
-  schema.TO_MULTI_LINK.filter(
-    any(multiLinkSchema.STRING_PROPERTY.equals('test'))
-  )
-);
+
+// -----------
+/*
+TODO: Fix the type issue for the following `and(...)` test
```

Comment:
@marikaner I discussed yesterday with @ZhongpinWang that I think the solution of this type issue is beyond this ticket and we should do it as a follow up so that we can get the majority of type tests back in the pipeline quickly.

Reply:
Makes sense.